dapps-profiling/src/profilings/liquidity-pool/setup.ts

```diff
@@ -0,0 +1,207 @@
+
+  const opexAccount = new StellarPlus.Account.DefaultAccountHandler({
+    network,
+  });
+
+  const issuerAccount = new StellarPlus.Account.DefaultAccountHandler({
+    network,
+  });
+
+  console.log("Opex: ", opexAccount.getPublicKey());
+  console.log("issuer: ", issuerAccount.getPublicKey());
+
+  const promises = [
+    opexAccount.friendbot?.initialize() as Promise<void>,
+    issuerAccount.friendbot?.initialize() as Promise<void>,
+  ];
+
+  await Promise.all(promises).then(() => {
+    console.log("Base accounts initialized!");
+  });
+
+  const opexTxInvocation = {
+    header: {
+      source: opexAccount.getPublicKey(),
+      fee: "10000000", //1 XLM as maximum fee
+      timeout: 30,
+    },
+    signers: [opexAccount],
+  };
+
+  const issuerTxInvocation = {
+    header: {
+      source: issuerAccount.getPublicKey(),
+      fee: "1000000", //0.1 XLM as maximum fee
+      timeout: 0,
+    },
+    signers: [issuerAccount],
+    feeBump: opexTxInvocation,
+  };
+
+  const opex = {
+    account: opexAccount,
+    transactionInvocation: opexTxInvocation,
+  };
+
+  const issuer = {
+    account: issuerAccount,
+    transactionInvocation: issuerTxInvocation,
+  };
+
+  return { opex, issuer };
+};
+
+export type CreateAssetsArgs = {
+  network: Network,
+  txInvocation: TransactionInvocation,
+  validationCloudApiKey?: string
+}
+
+export type CreateAssetsResult = {
+  assetA: SorobanTokenHandler,
+  assetB: SorobanTokenHandler,
+}
+
+export async function createAbundanceAsset({ network, txInvocation, validationCloudApiKey }: CreateAssetsArgs): Promise<CreateAssetsResult> {
+
+  const abundanceAssetWasm = await loadWasmFile(
+    "./src/dapps/soroban-token/wasm/soroban_token_contract.wasm"
+  );
+
+  const vcRpc = validationCloudApiKey
+    ? new StellarPlus.RPC.ValidationCloudRpcHandler(
+      network,
+      validationCloudApiKey
+    )
+    : undefined;
+
+  const sorobanTokenA = new SorobanTokenHandler({
+    network,
+    wasm: abundanceAssetWasm,
+    rpcHandler: vcRpc,
+  });
+
+  console.log("Uploading Soroban Token A WASM Files...");
+  await sorobanTokenA.uploadWasm(txInvocation)
+  console.log("deploy Soroban Token A...");
+  await sorobanTokenA.deploy(txInvocation)
+  console.log("initialize Soroban Token A...");
+  await sorobanTokenA.initialize({
+    admin: txInvocation.header.source,
+    decimal: 7,
+    name: "Token A",
+    symbol: "TAKA",
+    ...txInvocation
+  })
+
+  const sorobanTokenB = new SorobanTokenHandler({
+    network,
+    wasm: abundanceAssetWasm,
+    rpcHandler: vcRpc,
+  });
+
+  console.log("Uploading Soroban Token B WASM Files...");
+  await sorobanTokenB.uploadWasm(txInvocation)
+
+  console.log("Deploy Soroban Token B...");
+  await sorobanTokenB.deploy(txInvocation)
+
+  console.log("Initialize Soroban Token B...");
+  await sorobanTokenB.initialize({
+    admin: txInvocation.header.source,
+    decimal: 7,
+    name: "Token B",
+    symbol: "TAKB",
+    ...txInvocation
+  })
+
```

Comment:
It would be good to include something like this to check the order of the assets
```
if ((assetA.getContractId()) >= (assetB.getContractId())) {
    let tempAssetA = assetA;
    assetA = assetB;
    assetB = tempAssetA;
}
```